docs/reference/plugins.md

```diff
@@ -22,22 +22,48 @@ Siehe [MBMD](https://github.com/volkszaehler/mbmd) für ein Beispiel wie man Mod
 **Beispiel Lesen**:
 
 ```yaml
-source: mqtt
-topic: mbmd/sdm1-1/Power
-timeout: 30s # don't accept values older than timeout
-scale: 0.001 # floating point factor applied to result, e.g. for Wh to kWh conversion
+meters:
+- name: grid-meter
+  type: custom
+  power:
+    source: mqtt
+    topic: mbmd/sdm1-1/Power # MQTT topic auf dem der Wert empfangen wird
+    timeout: 30s # wie alt empfangene Wert maximal 
+                 # sein darf um berücksichtigt zu werden
+    scale: 0.001 # Konvertierungsfaktor um MQTT Wert auf W umzurechnen 
```

Comment:
Watt hat nix mit MQTT zu tun.

Reply:
In der urspr. Dokumentation steht

```
scale: 0.001 # floating point factor applied to result, e.g. for Wh to kWh conversion
```

Nach was muss ich den konvertieren ? Meinem Eindruck erwarte evcc für die UI, das alles Werte in W bzw. Wh erwartet wird. Der Kommentr is ja aber auch falsch, das es Wh in mWh umrechnen wuerde.